src/client/src/orangehrmCoreOAuthPlugin/pages/oauth/OAuthAuthorize.vue

```diff
@@ -19,27 +19,32 @@
  -->
 
 <template>
-  <oxd-text tag="h6">
-    Authorize <b>{{ clientName }}</b></oxd-text
-  >
-  <oxd-form
-    ref="authorizeForm"
-    method="GET"
-    :action="submitUrl"
-    @submit-valid="onSubmit"
-  >
-    <input name="authorized" :value="authorized" type="hidden" />
-    <div v-for="(value, name, index) in params" :key="name">
-      <input :name="name" :value="value" type="hidden" />
-      <p>{{ index }}. {{ name }}: {{ value }}</p>
-    </div>
-    <oxd-button
-      display-type="ghost"
-      :label="$t('general.cancel')"
-      @click="onCancel"
-    />
-    <submit-button :label="$t('general.submit')" />
-  </oxd-form>
+  <oxd-text v-if="errorType !== null" tag="h6">
```

Comment:
this screen is dummy screen for the consent screen, @devishke-orange will complete it